packages/backend/src/services/HealthService/HealthService.ts

```diff
@@ -44,8 +44,6 @@ export class HealthService {
             );
         }
 
-        const needsProject = !(await this.projectModel.hasProjects());
```

Comment:
we can remove all references to the project model in the health service

Reply:
I agree, but I'll be doing that on https://github.com/lightdash/lightdash/issues/2094#issuecomment-1125786147